content/releases/release-4.14.0.md

```diff
@@ -1,5 +1,6 @@
 ---
 date: 2025-08-19
+eol: 2026-08-18
```

Comment:
The EOL date was missing in the Camel 4.14 release. I hope I have set correctly, @davsclaus @oscerd?

Reply:
Ah thank you, yeah its 1 year so its correct